rest/call/devnet.go

```diff
@@ -0,0 +1,49 @@
+package call
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/tenderly/tenderly-cli/rest"
+	"github.com/tenderly/tenderly-cli/rest/client"
+	"github.com/tenderly/tenderly-cli/rest/payloads"
+)
+
+var _ rest.DevNetRoutes = (*DevNetCalls)(nil)
+
+type DevNetCalls struct{}
+
+func NewDevNetCalls() *DevNetCalls {
+	return &DevNetCalls{}
+}
+
+type SpawnRPCRequest struct {
+	Template string `json:"templateSlugOrId"`
+}
+
+func (rest *DevNetCalls) SpawnRPC(accountID string, projectID string, templateSlug string) (string, error) {
+	req := &SpawnRPCRequest{
+		Template: templateSlug,
+	}
+	reqJson, err := json.Marshal(req)
+	if err != nil {
+		return "", err
+	}
+
+	path := fmt.Sprintf("api/v1/account/%s/project/%s/devnet/container/spawn-rpc", accountID, projectID)
+	resp := client.Request(
```

Comment:
Can we add a way to specify the access key from the flag? 

This is mainly due to the nature of the CI.